ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     displayName: 'Install .NET Core 3.x SDK'
     inputs:
       packageType: 'sdk'
-      version: '3.1.100-preview3-014645'
+      version: '3.1.100'
       includePreviewVersions: true
```

Comment:
Do we still need includePreviewVersions as this is a full release?

Reply:
Good spot @jageall